package.json

```diff
@@ -4,10 +4,10 @@
   "description": "A server for the Sims dewire group",
   "main": "index.js",
   "scripts": {
-    "start": "concurrently --kill-others \"npm run mongodb\" \"npm run server\"",
-    "mongodb": "echo 'Please make sure MongoDB is installed' && mongod --dbpath data/db/ && mkdir -p logs/",
+    "start": "concurrently --kill-others \"node mkdirp.js\" \"npm run mongodb\" \"npm run server\"",
```

Comment:
Please replace

"start": "concurrently --kill-others \"node mkdirp.js\" \"npm run mongodb\" \"npm run server\"",
"mongodb": "echo 'Please make sure MongoDB is installed' && mongod --dbpath data/db/",

...with

"start": "concurrently --kill-others \"npm run mongodb\" \"npm run server\"",
"mongodb": "echo 'Please make sure MongoDB is installed' && node mkdirp.js && mongod --dbpath data/db/",

(mkdirp must be run before, not concurrently with, mongodb daemon)